libconfig/GlobalConfigure.h

```diff
@@ -60,6 +71,33 @@ class GlobalConfigure
     void setEVMSchedule(dev::eth::EVMSchedule const& _schedule) { m_evmSchedule = _schedule; }
     dev::eth::EVMSchedule const& evmSchedule() const { return m_evmSchedule; }
 
+
+    static uint32_t getVersionNumber(const std::string& _version)

```

Comment:
I think that the code below can satisfy you without move the `getVersionNumber` function 
```cpp
enum VERSION : uint32_t
{
    RC1_VERSION = 1,
    RC2_VERSION = 2,
    RC3_VERSION = 3,
    V2_0_0 = 0x02000000
};
```

Reply:
already fixed